kit/ForKit.cpp

```diff
@@ -746,6 +746,7 @@ int forkit_main(int argc, char** argv)
         const auto conf = std::getenv("COOL_CONFIG");
         config::initialize(std::string(conf ? conf : std::string()));
         EnableExperimental = config::getBool("experimental_features", false);
+        SigUtil::setIndirectionServerEnabled(!config::getString("indirection_endpoint.url", "").empty());
```

Comment:
Do we want and/or need this ? should a Kit be getting a SIGTERM and doing something - I think not; can we drop that ?